src/site/content/content.11tydata.js

```diff
@@ -2,6 +2,20 @@ const outputPermalink = require('../../build/output-permalink');
 
 module.exports = {
   eleventyComputed: {
+    tags: (data) => {
+      // Add the following tags if necessary
+      const originTrials = 'origin-trials';
+
+      if (!Array.isArray(data.tags)) {
+        data.tags = [];
```

Comment:
technically this clobbers tags if someone accidentally only set a single string, but I think that's incredibly unlikely.
🤷